Configuration/Settings.yaml

```diff
@@ -29,11 +29,21 @@ Neos:
           createParentDirectories: true
           maximumLogFileSize: 1048576
           logFilesToKeep: 1
+      updateLogger:
+        backend: Neos\Flow\Log\Backend\FileBackend
+        backendOptions:
+          logFileURL: '%FLOW_PATH_DATA%Logs/GoogleUpdate.log'
```

Comment:
I'd rather call the logger "jobIndexingLogger" and the logFile "JobIndexingLog", what do you think?

Reply:
*Updated: There is now one logger "jobIndexingLogger" and one logFile "JobIndexingLog"*